.github/workflows/main.yml

```diff
@@ -14,40 +15,31 @@ jobs:
       uses: actions/checkout@v2
       with:
         fetch-depth: 0
-
-    # set up python and node ----
     - name: Set up Python
       uses: actions/setup-python@v1
       with:
         python-version: 3.8
     - uses: actions/setup-node@v2
       with:
         node-version: '14'
-
-    # install dependencies ----
     - name: Install python packages
       run: |
         pip install -r requirements.txt
     - name: Install node packages
       run: npm install
       working-directory: website
-
-    # copy in reports ----
-
     - uses: google-github-actions/setup-gcloud@v0
       with:
         service_account_key: ${{ secrets.GCP_SA_KEY }}
         export_default_credentials: true
 
-    # development
+    # copy data ----
     - name: Copy development report data
-      if: ${{ github.ref == 'refs/heads/development' }}
+      if: ${{ github.event_name == 'push' }}
```

Comment:
Just to clarify. Any push to main will update the dev site, and a new tag will push to prod?

Reply:
Yes that's the ultimate goal.